### DISCUSSION TITLE
Counter Strike Restored support

### DISCUSSION THREAD

**Discussion by Papajak123y** [akiver/cs-demo-manager#949] [Q&A] [answered]
Hey, i'm an admin on a csgo project on a build from 2016. i wanted to use demo manager to make it easier to check demos for reported players but i cant get it to work. I dont think it will work with legacy since it's completely different build. is there a way to make it work on our build? The CSGO project im talking about Is Counter Strike Restored
it would be more convenient to discuss it on discord
my discord: 21papaj37

**Comment by akiver** [comment, marked as answer]
Checked on Discord, the app supports Counter Strike Restored, you have to:

- To analyze demos: change the demo's source to eBot since the servers use a custom eBot
- To watch demos: select the `csgo_legacy` branch to [watch CSGO demos](https://cs-demo-manager.com/docs/guides/playback#watching-csgo-demos), the last official CSGO build is able to play demos from Counter Strike Restored,
